modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -0,0 +1,1200 @@
+			\Elementor\Group_Control_Typography::get_type(),
+			[
+				'name' => 'cta_links_typography',
+				// TODO: add class selector when markup is done
+				// 'selector' => '{{WRAPPER}} .your-class',
+			]
+		);
+
+		$this->add_control(
+			'cta_links_show_border',
+			[
+				'label'        => esc_html__( 'Border', 'elementor' ),
+				'type'         => Controls_Manager::SWITCHER,
+				'label_on'     => esc_html__( 'Yes', 'elementor' ),
+				'label_off'    => esc_html__( 'No', 'elementor' ),
+				'return_value' => 'yes',
+				'default'      => 'no',
+			]
+			// TODO: add class selector when markup is done
+			// 'selectors' => [
+			// 	'{{WRAPPER}} .elementor-tab-title' => 'border-width: {{SIZE}}{{UNIT}};',
+			// 	'{{WRAPPER}} .elementor-tab-content' => 'border-width: {{SIZE}}{{UNIT}};',
+			// ],
+		);
+
+		$this->add_control(
+			'cta_links_border_width',
+			[
+				'label'      => esc_html__( 'Border Width', 'elementor' ) . ' (px)',
+				'type'       => Controls_Manager::SLIDER,
+				'size_units' => [ 'px' ],
+				'range'      => [
+					'px' => [
+						'min'  => 0,
+						'max'  => 10,
+						'step' => 1,
+					],
+				],
+				'condition'  => [
+					'cta_links_show_border' => 'yes',
+
+				],
+				// TODO: add class selector when markup is done
+				// 'selectors' => [
+				// 	'{{WRAPPER}} .elementor-tab-title' => 'border-width: {{SIZE}}{{UNIT}};',
+				// 	'{{WRAPPER}} .elementor-tab-content' => 'border-width: {{SIZE}}{{UNIT}};',
+				// ],
+			]
+		);
+
+		$this->add_control(
+			'cta_links_border_color',
+			[
+				'label'     => esc_html__( 'Border Color', 'elementor' ),
+				'type'      => Controls_Manager::COLOR,
+				'condition' => [
+					'cta_links_show_border' => 'yes',
+				],
+				// TODO: add class selector when markup is done
+				// 'selectors' => [
+				// 	'{{WRAPPER}} .elementor-tab-content' => 'border-bottom-color: {{VALUE}};',
+				// 	'{{WRAPPER}} .elementor-tab-title' => 'border-color: {{VALUE}};',
+				// ],
+			]
+		);
+
+		$this->add_control(
+			'cta_links_corners',
+			[
+				'label'   => esc_html__( 'Corners', 'elementor' ),
+				'type'    => Controls_Manager::SELECT,
+				'default' => 'round',
+				'options' => [
+					'round'   => esc_html__( 'Round', 'elementor' ),
+					'rounded' => esc_html__( 'Rounded', 'elementor' ),
+					'sharp'   => esc_html__( 'Sharp', 'elementor' ),
+				],
+				// TODO: add class selector when markup is done
+				// 'selectors' => [
+				// 	'{{WRAPPER}} .your-class' => 'border-style: {{VALUE}};',
+				// ],
+			]
+		);
+
+		$this->add_control(
+			'hr',
+			[
+				'type' => Controls_Manager::DIVIDER,
+			]
+		);
+
+		$this->add_control(
+			'cta_links_padding',
+			[
+				'label'      => esc_html__( 'Padding', 'elementor' ),
+				'type'       => Controls_Manager::DIMENSIONS,
+				'size_units' => [ 'px', '%', 'em', 'rem' ],
+				'default'    => [
+					'top'      => 2,
+					'right'    => 0,
+					'bottom'   => 0,
+					'left'     => 0,
+					'unit'     => 'px',
```

Comment:
It is a good idea to add them to the SASS file

Reply:
@rodolphebertozzo could you elaborate a little please. Thank you